### DISCUSSION TITLE
how to start translating content to new language?

### DISCUSSION THREAD

**Discussion by F4R4N** [mdn/mdn-community#577] [Content] [answered]
I am trying to translate MDN contents into Persian. based on the guide, I need Persian reviewers and I can find one or two. but when should I open a pull request? in which stage? should I translate the whole content, one part? please help me out with this.
Thanks

**Comment by Rumyra** [comment, marked as answer]
Hi @F4R4N - unfortunately Persian is not one of our top tier locales, so we won't be supporting it. There's more information here https://hacks.mozilla.org/2021/03/mdn-localization-in-march-tier-1-locales-unfrozen-and-future-plans/  which I hope helps explain it 🙏

**Comment by pepelsbey** [comment]
Closing as answered/settled.
